### DISCUSSION TITLE
Cannot change server on open connection

### DISCUSSION THREAD

**Discussion by wardellcastles** [mitmproxy/mitmproxy#5682] [Q&A]
I am getting a runtime error in **connection.py**.  The message is "Cannot change server.{name} on open connection".   Runtime Error: Cannot change server.via on open connection.
This module is part of a MobSF installation performing a Dynamic Analysis   
I am unsure what servers are attempting to be changed.  Everything is running on my local machine.
How can I resolve this?

**Comment by Prinzhorn** [comment]
> This module is part of a MobSF installation performing a Dynamic Analysis

Sounds like a bug in whatever MobSF is. If you cannot reproduce it with vanilla mitmproxy, then we can't help you.
